tests/wpt-harness/tests.json

```diff
@@ -0,0 +1,162 @@
+
```

Comment:
Just want to make sure that these files should point to the `.js` and not the `.json` correct? 😄  (I've never used the wpt toests, so I genuinely don't know 😅 )

Reply:
yeah, these are the actual test files to run. The `json` files are the expectations we store for the test runs, and we derive their paths from the `js` file path. Could go the other way in theory, but since we can run tests that don't yet have expectations attached, that'd be a bit weird :)